### DISCUSSION TITLE
Curiosity of New PhysX Physics Accuracy

### DISCUSSION THREAD

**Discussion by zoctipus** [NVIDIA-Omniverse/PhysX#274] [General]
I am curious of PhysX Physics Accuracy, I have done some research but couldn't find many answer. Many comment online (especially earlier) says that PhysX is a game engine is optimized for speed not suitable for high fidelity physics simulation. In the same time, I am also hearing that Nvidia's enphasis in moving into robotic field and increased physical accuracy and updating PhysX5’s physic calculation datatype precision and solver. I wonder what would you say about it? What is improvement is PhysX's rigidbody physix compare to previous version? Will it still just be a game engine or can it be used for higher fidelity industry or even scientific research

**Comment by preist-nvidia** [comment]
Hi @zoctipus - it is difficult to make general accuracy statements or assurances about an all-purpose simulator as there are many variables in the simulation scene setup and settings that may affect accuracy. We do have users that achieve high-enough accuracy for robotics and industrial simulations, but what can be achieved depends on the specific simulation and what level of fidelity is required.

**Reply by Zeblote** [reply]
Do you have any news on double coordinates being supported in PhysX?

**Reply by preist-nvidia** [reply]
Hi @Zeblote ! No news, we have no plans to support double precision at the moment.
